keyring.php

```diff
@@ -263,6 +260,10 @@ static function debug( $str, $level = KEYRING__DEBUG_NOTICE ) {
 		error_log( "Keyring: $str" );
 	}
 
+	/**
+	 * @deprecated
+	 * @return boolean
+	 */
 	static function is_service( $service ) {
 		if ( is_object( $service ) && is_subclass_of( $service, 'Keyring_Service' ) ) {
```

Comment:
This could probably just be:

```
return is_object( $service ) && is_subclass_of( $service, 'Keyring_Service' );
```

But as we're deprecating it then it's not really worth changing.